README.md

```diff
@@ -12,7 +12,7 @@ For information on data models and to view the entire DAG check out our dbt [Doc
 
```

Comment:
The link to the knowledge base in the ReadMe is broken because some of the sections were moved/changed.  It should be updated to go to [this](https://thetuvaproject.com/docs/readmissions) page.

Reply:
fixed